schemas/telemetry/account-ecosystem/account-ecosystem.4.schema.json

```diff
@@ -74,6 +170,9 @@
           },
           "type": "array"
         },
+        "reason": {
+          "type": "string"
+        },
```

Comment:
I also add payload.reason here, which is currently getting sent (and stored in additional_properties) and is quite standard for common pings.